### PR TITLE
Track OS X test worker types

### PR DESCRIPTION
**Pull request by gregarndt**

### CODE REVIEW

File: src/collector/pending.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ const NONPROVISIONED_WORKERTYPES = [
   'scriptworker-prov-v1.pushapk-v1',
   'scriptworker-prov-v1.signing-linux-v1',
   'signing-provisioner-v1.signing-worker-v1',
+  'releng-hardware/gecko-t-osx-1010',
```
Comment (djmitche) [inline src/collector/pending.js]: `.` instead of `/`

Comment (gregarndt) [reply src/collector/pending.js]: doh, thanks for catching that.  Fixed in e35bb31

**Comment by gregarndt** [comment]
So I'm hesitant rolling this in because of the different pending time requirements between aws provisioned machines and physical hardware.  Especially if we ever want to take error budget seriously.  having a 30 minute pending threshold for aws might be reasonable, but it's almost never reasonable for OS X.  Most days we hit a few hour backlog on OS X.

**Comment by djmitche** [comment]
Yeah, even with AWS this wasn't really the right way to measure things.

**Pull request closed without merging**